### PR TITLE
PCHR-2989: Update ContactSummary CSS

### PR DESCRIPTION
**Pull request by reneolivo**
## Overview
This PR updates the contact summary CSS so it includes the new colours added in Shoreditch.

## Before
civihr_staff compucorp co uk hr17 9000 6

## After
civihr_staff compucorp co uk hr17 9000 7

## Technical Details
Since the Leave Widget referenced the colours defined in Shoreditch and they were updated, the Contact Summary needed to be updated as well by running `gulp sass`.

## Comments
Ran backstop-config (the repo) on contact page and the differences are in colour only (as expected).

### CODE REVIEW

**Comment by reneolivo** [comment]
## New Changes

### Heatmap text colours

The heatmap now has black text colours instead of white, except for the "High" (red) value, which remains white.

#### Before
civihr_staff compucorp co uk hr17 9000 11

#### After
civihr_staff compucorp co uk hr17 9000 10

### L&A Shoreditch update

#### Before
anim
Notice how the blue text changes colour after going to the absence tab and going back to the contact summary.

civihr_staff compucorp co uk hr17 9000 12


#### After

File: contactsummary/scss/contactsummary.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
+@import "SCSSROOT/org.civicrm.shoreditch/scss/civicrm/variables";
```
Comment (AkA84) [inline contactsummary/scss/contactsummary.scss]: why is this necessary? nothing in the civicrm side of Shoreditch should be used by apps that use bootstrap-based markup
